### ISSUE TITLE
vim cannot highlighted.

### ISSUE THREAD

**Issue by HisenseLiu** [kingToolbox/WindTerm#440]
When I open the file with vim, the keywords in the file are not highlighted. But when I use the more or tail commands it does work fine. Does windterm not adapt to <line editor>?

**Comment by kingToolbox** [comment]
Yes, that's how it's designed, because `vim` usually has its own `color scheme`, so `WindTerm` doesn't colorize the content of `vim` again.
